src/rinterface_extra.c

```diff
@@ -3172,9 +3172,13 @@ igraph_error_t R_SEXP_to_sparsemat(SEXP pakl, igraph_sparsemat_t *akl) {
   SEXP p=GET_SLOT(pakl, Rf_install("p"));
   SEXP x=GET_SLOT(pakl, Rf_install("x"));
 
+  igraph_vector_int_t ci, cp;
+  IGRAPH_CHECK(R_SEXP_to_vector_int_copy(i, &ci));
+  IGRAPH_CHECK(R_SEXP_to_vector_int_copy(p, &cp));
```

Comment:
This memory will need to be released.

Reply:
Yes, it should. I have created draft to discuss it. 
Is it sounds okay to make a copy for all vectors in `cs_igraph_sparse` and use `igraph_sparsemat_destroy` after conversion?

Reply:
Anyway we need to copy 2 of vectors `i` and `p` due to the `number()` to `igraph_integer_t` conversion. Maybe it will just good enough to make `R_SEXP_to_sparsemat_copy`?